manual/googlechrome-extensions/googlechrome-github-expandinizr/tools/install_x64.reg

```diff
@@ -0,0 +1,2 @@
+[HKEY_LOCAL_MACHINE\SOFTWARE\Wow6432node\Google\Chrome\Extensions\cbehdjjcilgnejbpnjhobkiiggkedfib]
```

Comment:
Found the problem, registry keys are case sensitive.
Change `Wow6432node` to `Wow6432Node`.
An upper case `N` is all that is needed.